tokio/tests/rt_metrics.rs

```diff
@@ -82,20 +82,47 @@ fn blocking_queue_depth() {
 }
 
 #[test]
-fn active_tasks_count() {
+fn num_active_tasks() {
```

Comment:
Seems like this test is missing an assert for the number going back down afterwards.